client/src/controllers/MessagesController.ts

```diff
@@ -0,0 +1,46 @@
+import { Controller } from '@hotwired/stimulus';
+
+export class MessagesController extends Controller<HTMLFormElement> {
+  static classes = ['newMessage', 'show'];
```

Comment:
```suggestion
  static classes = ['added', 'show'];
```

Let's call this the added class, it's simpler and makes more sense, plus avoids putting the word message twice in the Dom.